x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/utils/get_mode_display_text.test.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  ADMIN,
+  PRIVATE,
+  READ_ONLY,
+  READ_WRITE,
+  SEARCH,
+  SEARCH_DISPLAY,
+  WRITE_ONLY,
+} from '../constants';
```

Comment:
This is not a change request, just me thinking out loud cont'd. This is my personal preference/longings and I think the tests are just fine as-is, but I would have super loved to read the tests with their actual input/output strings vs constants vars to help me understand:

1. What the API tokens coming in from our server actually look like
2. What the output in the UI actually looks like 

Having the 'real' text personally helps me a lot with visualizing what the code is doing in & tying it all together in my head. I know it's a bit more of a pain if the strings/constants ever change but just IMO the find&replace is worth it for developer understanding.

That being said though, not a change request, just a comment, so if you feel strongly about vars in test or just don't want to bother, def feel free to ignore this

Reply:
I agree with you there. I had wondered about the same thing. I'll make this change.

Reply:
9e6b0c4

Reply:
Love it!! 🎉